core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -244,6 +244,21 @@ oppia.controller('StateInteraction', [
                   stateCustomizationArgsService.displayed).length > 0);
               }
 
+              $scope.areCustomizationArgsValid = function() {
+                if (!stateCustomizationArgsService.displayed) {
+                  return false;
+                }
+                var validationService =
+                  $injector.get(
+                    INTERACTION_SPECS[
+                      $scope.stateInteractionIdService.displayed].id +
+                    'ValidationService');
+                var warningsList =
+                  validationService.getCustomizationArgsWarnings(
+                    stateCustomizationArgsService.displayed);
+                return warningsList.length == 0;
```

Comment:
Yep! We generally use === and !== (otherwise I think you'll fail the frontend linter).